angular2/src/angular2TextMask.ts

```diff
@@ -74,5 +74,5 @@ export class MaskedInputDirective implements OnInit, ControlValueAccessor{
   exports: [MaskedInputDirective],
   imports: [CommonModule]
 })
-export class TextMaskModule {
-}
+export class TextMaskModule {}
+export {conformToMask} from 'text-mask-core/dist/textMaskCore'
```

Comment:
@PerfectPixel If we implement it as a pipe in Angular, we'll need to add more documentation and write more code (in Angular 1 and 2)...and I don't see what additional benefits being a pipe provides other than the cool pipe syntax I guess :)